WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -40,7 +48,9 @@ struct WebView: UIViewRepresentable {
     }
 
     func updateUIView(_ uiView: WKWebView, context: Context) {
-
+        if let device = previewDevice {
+            uiView.reload()
+        }
```

Comment:
To keep this check generic, we can add a property `shouldReloadOnUpdate` and set its value to `true` only for theme preview.

Reply:
This is added in https://github.com/woocommerce/woocommerce-ios/pull/11452/commits/f36885e499f8ede9bf409d7bdb8ac184d24e541c